core_brain/shadow_run.py

```
@@ -482,6 +482,9 @@ def run_shadow(
     fetch_books: Optional[Callable] = None,
     interval: float = 5.0,
     funder: Optional[str] = None,
+    run_id: Optional[str] = None,
+    sleep_fn: Optional[Callable[[float], None]] = None,
+    cfg: Optional[MakerConfig] = None,
```

Ruff (0.16.2)

[error] Undefined name `MakerConfig`

(F821)